# 机器人位置数据

Header header

# X轴
float64 x
# Y轴
float64 y
# Z轴
float64 z
# 枪口YAW轴,单位弧度
float64 shooterYaw